docker-compose.yml

```diff
@@ -1,26 +1,26 @@
 version: '3.7'
 
 services:
-  yarn:
-    image: front-end-monorepo_dev:latest
-    build:
-      context: ./
-      target: bootstrap
-    entrypoint:
-      - "yarn"
-    command: ["test:ci"]
-  fe-project:
-    image: front-end-monorepo_prod:latest
-    build:
-      context: ./
-      target: production-apps
-    entrypoint:
-      - "yarn"
-      - "workspace"
-      - "@zooniverse/fe-project"
-    command: ["start"]
-    ports:
-      - "3000:3000"
+  # yarn:
```

Comment:
This is very much a wip for quick testing, i'll reinstate these services once folks have tested locally and are happy with the solution